src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMProbe.java

```diff
@@ -57,8 +58,12 @@
     private final String name;
     private transient boolean open = true;
 
-    public GitHubSCMProbe(GitHub github, GHRepository repo, SCMHead head, SCMRevision revision) {
-        this.gitHub = github;
+    public GitHubSCMProbe(String apiUri,
```

Comment:
Will this affect configurations or anything? I don't see any public consumers of this class, and it isn't an `@DataBoundConstructor` or anything, so it seems safe. Maybe worth adding `@Restricted(NoExternalUse.class)` to either the class or the constructor if it should never be used by another plugin.

Reply:
@dwnusbaum 
Is the class being package-private not enough?

Reply:
Oops yeah that's fine, I thought it was public.